cms/templates/container.html

```diff
@@ -176,7 +176,15 @@ <h3 class="sr">${_("Page Actions")}</h3>
                             <span class="action-button-text">${_("Preview")}</span>
                         </a>
                     </li>
+
                 % else:
+                    % if is_library_xblock:
+                    <li class="action-item action-save nav-item">
+                        <a href="#" class="button button-save action-button save-button is-hidden">
+                            <span class="action-button-text">${_("Save changes")}</span>
+                        </a>
+                    </li>
+                    % endif
```

Comment:
I set up library_content block with manual==True and shuffle==False, went to the children page, and tried selecting children by checking boxes, but I don't see any "Save" button, so I can't save my selection.

![image](https://github.com/openedx/edx-platform/assets/3628148/e18df951-1e3c-4b9f-a019-eaae61c8012a)

Reply:
It might be you have to remake static assets/refresh. There is some client side JS in the PR, but I will also take a look.

Reply:
Confirmed. Updating static assets does the trick.

Reply:
Ohp, you're right!